app/jobs/catalog_job.rb

```diff
@@ -28,8 +28,12 @@ def update_registry!(catalog)
     # In this loop we will create/update all the repos from the catalog.
     # Created/updated repos will be removed from the "repos" array.
     catalog.each do |r|
-      cou   = Repository.create_or_update!(r)
-      dangling_repos = dangling_repos.delete_if { |re| re == cou.id }
+      if r["tags"].nil?
+        Rails.logger.debug "skip upload not finished repo #{r['name']}"
```

Comment:
Can you change that to `Rails.logger.debug "skip upload not finished repo #{r["name"]}"` otherwise the code styling checks will keep failing.